### ISSUE TITLE
[Chore] Use a UI library for consistency

### ISSUE THREAD

**Issue by malparty** [ashiqurrah7/nimble-be#19]
## Issue

The current UI is not consistent and lacks accessibility standards:

- The input fields have no style
- The HTML code does not leverage semantical elements (e.g. `nav`, `header`, `main`, `section`, etc...)
- The project uses components from `react-bootstrap` (which are rendered with the Bootstrap classes like `btn btn-primary`) but does not import the Bootstrap style
- Pressing `enter` in the login/register form does not submit the form
- Overall the screen space is not optimized, using pre-existing components (e.g. Bootstrap cards or tables) could help display more information like keyword name, statistics, uploaded date, etc... 

## Expected

This project does not require extensive accessibility or outstanding design, but it requires a consistent UI and some thoughts about the UX.
